paddle/fluid/framework/operator.h

```diff
@@ -104,6 +104,27 @@ constexpr char kEnableCacheRuntimeContext[] = "@ENABLE_CACHE_RUNTIME_CONTEXT@";
 constexpr char kAllKernelsMustComputeRuntimeShape[] =
     "@ALL_KERNELS_MUST_COMPUTE_RUNTIME_SHAPE@";
 
+struct VarInfo {
```

Comment:
```suggestion
struct VarMetaInfo {
```
OP的`name`、`dtype`和`place`用`MetaInfo`命名更贴切，也与phi下`InferMeta`中的`Meta`语义一致。但nw_executor_defs.h中已有一个`VariableMetaInfo`，两者容易混淆。可以考虑这里先用`VarMetaInfo` 命名，后续给`VariableMetaInfo`换个名字。